Sources/Components/SuggestShippingView/SuggestShippingView.swift

```diff
@@ -0,0 +1,139 @@
+import Combine
+import FinniversKit
+
+public final class SuggestShippingView: UIView {
+    private var viewModel: SuggestShippingViewModel!
+    private var cancellables = Set<AnyCancellable>()
+    private var buttonCancellable: AnyCancellable?
+
+    private let imageHorizontalInset: CGFloat = .spacingS
+
+    private let horizontalContainer: UIStackView = {
+        let stackView = UIStackView(axis: .horizontal)
+        stackView.alignment = .center
+        return stackView
+    }()
+
+    private let containerView: UIStackView = {
+        let stackView = UIStackView(axis: .vertical)
+        stackView.spacing = .spacingM
+        return stackView
+    }()
+
+    private let textContainer: UIStackView = UIStackView(axis: .vertical)
+
+    private lazy var title: Label = {
+        let label = Label(style: .captionStrong)
+        label.numberOfLines = 0
+        label.textAlignment = .center
+        return label
+    }()
+
+    private lazy var message: Label = {
+        let label = Label(style: .caption)
+        label.numberOfLines = 0
+        label.textAlignment = .center
+        return label
+    }()
+
+    private lazy var suggestShippingbutton: Button = {
+        let button = Button(style: .suggestShippingStyle)
+        button.contentEdgeInsets = UIEdgeInsets(
+            top: .spacingS + .spacingXS,
+            leading: .spacingXL,
+            bottom: .spacingS + .spacingXS,
+            trailing: .spacingXL + imageHorizontalInset
+        )
+        button.titleEdgeInsets = UIEdgeInsets(
+            top: 0,
+            leading: imageHorizontalInset,
+            bottom: 0,
+            trailing: -imageHorizontalInset
+        )
+        button.translatesAutoresizingMaskIntoConstraints = false
+        let shippingTruck = UIImage(named: .shippingTruck)
+        let darkShippingTruck = shippingTruck.withTintColor(.textAction)
+        shippingTruck.imageAsset?.register(darkShippingTruck, with: .init(userInterfaceStyle: .dark))
```

Comment:
Isn't it possible just to change the tintColor of `button.imageView`?

Reply:
I think it's the correct way of doing it similar to e.g. setting a label's tint color for both light and dark mode. The other way we would need to listen to trait changes.